WiseBuy/APIs/APIHelper.h

```diff
@@ -0,0 +1,29 @@
+//
+//  APIHelper.h
+//  WiseBuy
+//
+//  Created by Mai Ngo on 7/21/22.
+//
+
+#import <Foundation/Foundation.h>
+#import "Item.h"
+#import "Deal.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol APIDelegate <NSObject>
+
++ (void)fetchDeals: (NSString *)barcode;
+
+@end
+
+@interface APIHelper : NSObject
+@property (weak, nonatomic) id<APIDelegate> delegate;
```

Comment:
Why do you have the delegate property here? It looks like it is never used. As well, you only have class methods here, so any of the object properties will not be accessible from those

Reply:
will delete it